packaging/debian/service

```diff
@@ -4,7 +4,7 @@ Before=
 After=network.target
 [Service]
```

Comment:
Should the service be started on installation?

I get the following message:
```
stackable-agent.service is a disabled or a static unit not running, not starting it.
```

Reply:
It was a conscious decision "back in the day" to not enable services at installation, as we wanted to give people the chance to perform additional operations like template config etc. before starting.